webdriver/tests/element_send_keys/send_keys.py

```diff
@@ -71,3 +71,29 @@ def test_stale_element(session, inline):
 
     response = element_send_keys(session, element, "foo")
     assert_error(response, "stale element reference")
+
+@pytest.mark.parametrize("value",[
+    [u'\ue018', ";"],
+    [u'\ue019', "="],
+    [u'\ue024', "*"],
+    [u'\ue025', "+"],
+    [u'\ue027', "-"],
+    [u'\ue028', "."],
+    [u'\ue029', "/"],
+    [u'\ue01a', "0"],
+    [u'\ue023', "9"]
```

Comment:
Question: What about all the other numbers? Also it misses `\ue026`

Reply:
I didn't add all the values because wpt times out the test file if there are a lot of parametrised values. I can raise an issue but I can't solve that issue in this test suite.

Reply:
Ah that one. Now I remember. Do you actually have a link to such a job in the wpt CI? I would be interested in. But yes it's fine to file an issue and add a TODO comment here with a link to that issue.

Reply:
Do you have such a link to a broken job in CI?

Reply:
I didn't put it through CI. It was timing out locally. I assumed it would do the same locally as on the CI

Reply:
As discussed yesterday the underlying problem with Chrome is gone and personally I also do not see any timeout when running way more parameterized tests with Chrome. So I would suggest that we do not land the patch in a rush but get coverage for all the normalized keys.

I'm happy to update the PR in case you don't have the time for.

Reply:
Ok, I can see that Chrome and also Firefox is running into a timeout. When I have a look at the trace logs at least for Firefox locally I can see that there is not that much of an improvement that we can do beside the following:

1. Use a `# META: timeout=long` line at the very top which in this case would be fine to do.
2. For tests where we can combine characters lets do it so that we can safe the time as needed for setup/teardown and especially loading the test page.
3. Think about actually loading real data URIs instead of inline docs which will still be routed through wptserve. 

Especially 2) will help a lot here given that we spend 50% of the test's time in setup/teardown.